packages/slice-machine/src/legacy/lib/models/common/widgets/Link/components.tsx

```diff
@@ -125,6 +136,44 @@ export function Variants({
           <Text color="grey11">{switchLabel}</Text>
         </Box>
       </Box>
+      {enabled && (
+        <Box
+          border={{ bottom: true }}
+          padding={12}
+          flexDirection="column"
+          gap={8}
+        >
+          <Text variant="h5" color="grey11">
+            {optionsTitle}
+            {Boolean(error) && (
+              <Text variant="inherit" color="tomato9">{` ${error}`}</Text>
+            )}
+          </Text>
+          {variants?.map((variant, position) => (
+            <Box
+              key={position}
+              backgroundColor="white"
+              padding={{ inline: 12, block: 8 }}
+              flexDirection="column"
+              gap={8}
+              border
+              borderRadius={4}
+            >
+              <TextInput
+                placeholder="Variation option (e.g. Primary)"
```

Comment:
I'm not sure if this was changed intentionally, but in the the design, the placeholder is different.

```suggestion
                placeholder="Enter your variant name (eg. Primary)"
```

Reply:
i changed it intentionally, i'll check with come all copies before releasing